### PR TITLE
Password Change

### PR DESCRIPTION
**Pull request by rebbrunner**
- Adds better password validation
- Add password change option to dropdown menu

### CODE REVIEW

**Comment by rebbrunner** [comment]
fixes #35 fixes #39

File: website/templates/website/layout.html [hunk 1]
```diff
@@ -59,6 +59,8 @@
                         <ul class="dropdown-menu">
                             <li class="text-center">Signed in as <strong>{{ user.username }}</strong></li>
                             <li role="separator" class="divider"></li>
+                            <li><a href="{% url 'website:project_selection' %}">Project Selection</a></li>
```
Comment (psybers) [inline website/templates/website/layout.html]: this is below now - remove this line

File: website/templates/website/layout.html [hunk 2]
```diff
@@ -59,6 +59,8 @@
...
+                            <li><a href="{% url 'website:password_change' %}">Change Password</a></li>
```
Comment (psybers) [inline website/templates/website/layout.html]: lets move change password under 'your profile'

File: website/templates/website/password_change.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{% extends 'website/layout.html' %}
+{% block title %}Edit Password{% endblock %}
```
Comment (psybers) [inline website/templates/website/password_change.html]: 'Change password'

File: website/views.py [hunk 4]
```diff
@@ -42,17 +42,11 @@ def get(self, request):
...
-            password = form.cleaned_data['password']
-            user.set_password(password)
-            user.save()
-            user = authenticate(request, username=username, password=password)
-            if user is not None:
-                if user.is_active:
-                    login(request, user)
-                    return redirect('website:index')
-            return render(request, self.template_name, { 'form' : form })
+            form.save()
```
Comment (psybers) [inline website/views.py]: so after a register we no longer log them in?

**Review by psybers — CHANGES_REQUESTED** [review]

**Review by psybers — APPROVED** [review]

**Pull request merged**